server/command.go

```diff
@@ -87,13 +87,17 @@ func (p *Plugin) validateCommand(action string, parameters []string) string {
 func (p *Plugin) executeCommandPreview(teamName string, args *model.CommandArgs) {
 	found := false
 	for _, message := range p.getWelcomeMessages() {
-		if message.TeamName == teamName {
-			if err := p.previewWelcomeMessage(teamName, args, *message); err != nil {
-				p.postCommandResponse(args, "error occurred while processing greeting for team `%s`: `%s`", teamName, err)
-				return
+		var teamNamesArr = strings.Split(message.TeamName, ",")
+		for _, item := range teamNamesArr {
+			messageTeamName := strings.Trim(item, "\t")
+			if messageTeamName == teamName {
+				p.postCommandResponse(args, "%s", teamName)
+				if err := p.previewWelcomeMessage(teamName, args, *message); err != nil {
+					p.postCommandResponse(args, "error occurred while processing greeting for team `%s`: `%s`", teamName, err)
+					return
+				}
+				found = true
```

Comment:
```suggestion
		for _, name := range teamNamesArr {
			tn := strings.Trim(name, "\t")
			if tn == teamName {
				p.postCommandResponse(args, "%s", teamName)
				if err := p.previewWelcomeMessage(teamName, args, *message); err != nil {
					p.postCommandResponse(args, "error occurred while processing greeting for team `%s`: `%s`", teamName, err)
					return
				}
				found = true
```

Reply:
4/5 `s/item/name/` for more clarity on the variable name.
1/5 `s/messageTeamName/tn/` for for simplicity

Reply:
sure, ill look into this today and update the pr. Sorry for the earlier delays.